### PR TITLE
Refactor some redunant code && Replace some UT by UtFrameUtils

### PR DESCRIPTION
**Pull request by WingsGo**
This CL have no logic changed, just do some code refactor and use new UtFrameWork to replace some old UT.

### CODE REVIEW

**Review by imay — COMMENTED** [review]
LGTM, 
I think @morningman should review this PR

File: fe/src/main/java/org/apache/doris/backup/RestoreJob.java [hunk 1]
```diff
@@ -806,7 +739,54 @@ private void checkAndPrepareMeta() {
         // No log here, PENDING state restore job will redo this method
         LOG.info("finished to prepare meta and send snapshot tasks, num: {}. {}",
                  batchTask.getTaskNum(), this);
-        return;
+    }
+
+    private boolean backupReplicasNotEqual(PartitionInfo localPartInfo, Partition localPartition, Table localTbl,
```
Comment (morningman) [inline fe/src/main/java/org/apache/doris/backup/RestoreJob.java:744]: I think the part of `generate file mapping` can be a separated method not including in this method.
Or the name of this method can be confused.

Comment (WingsGo) [reply fe/src/main/java/org/apache/doris/backup/RestoreJob.java:744]: can we change the method name to `genFileMappingWhenBackupReplicasEqual`?

Comment (morningman) [reply fe/src/main/java/org/apache/doris/backup/RestoreJob.java:744]: Sorry for review delay. This name looks good to me.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**